enzyme/CMakeLists.txt

```diff
@@ -70,17 +71,25 @@ message("CMAKE_PREFIX_PATH " ${CMAKE_PREFIX_PATH})
 
 find_package(LLVM REQUIRED CONFIG)
 
+if (LLVM_ENABLE_PLUGINS)
+  message("LLVM_ENABLE_PLUGINS ON")
+else()
+  message("LLVM_ENABLE_PLUGINS OFF")
+  message("ENZYME_NO_PLUGINS ON")
+  set(ENZYME_NO_PLUGINS ON)
+endif()
+
```

Comment:
we should probalby do a check that at least one of ENZYME_ENABLE_PLUGINS and ENZYME_EXTERNAL_SHARED_LIB is set to ON otherwise this won't build anything and is likely a bug